src/components/app_catalog/app_detail/InstallAppForm.js

```diff
@@ -50,6 +54,15 @@ const InstallAppForm = props => {
         validationError={props.valuesYAMLError}
         value={props.valuesYAML}
       />
+
+      <FileInput
+        description='Sensitive configuration can be uploaded separately as a secret.'
+        hint={<React.Fragment>&nbsp;</React.Fragment>}
```

Comment:
I think we should discuss if we'll use `<React.Fragment />` or `< />` from now on, we kind of have a mix of both right now

Reply:
Let's use `< />` only. 

I'll fix that in a new PR.

ping @ograu @ubergesundheit

Reply:
`<></>` is generally nicer than `<React.Fragmen></React.Fragment>`! I like this decision. There one exception I can think of when returning fragments from `.map`. Here using `<React.Fragment>` seems the way to go to add the required `key=` attribute for iteration.

Reply:
I like both. Happy with whatever you decide.